core/sr-io/without_std.rs

```diff
@@ -385,6 +385,20 @@ pub mod ext {
 		/// - nonzero otherwise.
 		fn ext_submit_transaction(data: *const u8, len: u32) -> u32;
 
+		/// Returns information about the local node's network state.
+		///
+		/// # Returns
+		///
+		/// - The encoded `Result<offchain::LocalNetworkState, ()>`.
```

Comment:
I think it is worth noting that the ownership of the returned buffer is transferred to the runtime code and thus the runtime is reponsible for freeing it. Also it would useful to note that this is always a properly allocated pointer (and e.g. cannot be NULL), so the runtime code can always rely on it.

Can we have a description on what is `msg_len`, what does the message mean in this context, is the ownership of this pointer is transferred or just lended, etc.